webdriver-spec.html

```diff
@@ -5455,9 +5457,9 @@
     property, which has the possible
     values <code>"mouse"</code>, <code>"pen"</code>,
     and <code>"touch"</code>, a <code>primary</code> property which is
-    a boolean, a pressed property which is a set of integers
-    an <code>x</code> property which is an unsigned integer and
-    a <code>y</code> property which is an unsigned integer.
+    a boolean, a <code>pressed</code> property which is a set of
+    integers an <code>x</code> property which is an unsigned integer
+    and a <code>y</code> property which is an unsigned integer.
```

Comment:
It’s not clear whether `pressed` is a set of integers, or if `x` and `y` are separate properties associated with _pointer input source_.